labview-interop/src/memory.rs

```diff
@@ -195,21 +228,21 @@ mod lv_owned {
     ///);
     ///```
     #[repr(transparent)]
-    pub struct LvOwned<T: ?Sized>(UHandle<T>);
+    pub struct LvOwned<'a, T: ?Sized>(pub(crate) UHandle<'a, T>);
```

Comment:
As we spoke about on the issue. Please make the inner UHandle 'static and then we don't need the lifetime on LvOwned which simplifies the rest of the Owned API.

Reply:
This will take me more time.
Last time I added the lifetime annotation to UHandle and then went with the linter and added lifetimes everywhere needed to make the linter happy.

When I now try to make the LvOwned internal UHandle static I will need to refactor ToOwned and clone_... It needs to be clear that after cloning the lifetime has become static. ToOwned seems to support that, but I haven't been successful in expressing my intent in correct syntax...

Reply:
I understand. This is where the `new_unsized` method should help. That should be able to create the handle with the `'static` lifetime and then pass it to the initialisation function to do the copy.

The owned string shows an example of how to call this at https://github.com/WiresmithTech/Rust-LabVIEW-Interop/blob/50060a63374659bd34391db3cb5317b7760c93f1/labview-interop/src/types/string.rs#L230

Reply:
That was not my problem :)

I either run into trait bounds, lifetimes or type incompatibilies in to_owned()... something seems to just line up.